static/css/style.css

```diff
@@ -113,4 +112,126 @@ a {
   margin-bottom: 2rem;
   font-family: 'Roboto', sans-serif;
   font-size: 1.5rem;
+}
```

Comment:
Hi Marina,

It looks like the CSS file has been doubled up? Can you check this out?
We already have an import at the top, and it looks like there is some overlap here.